XCUITests/FxScreenGraph.swift

```diff
@@ -801,7 +801,7 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
             for _ in 1...4 {
                 introScrollView.swipeLeft()
             }
-            app.buttons["Sign in to Firefox"].tap()
+            app.buttons["Turn on Sync..."].tap()
```

Comment:
Just curious, I have not had the chance to look at this, is there an ID for this element, so that we don't have this issue in the future if the string changes again...

Reply:
@SimonBasca this is the place where you can [add an ID for this button ](https://github.com/mozilla-mobile/firefox-ios/blob/4730536f913bae9e4817bd1fb03832c2bae29ba1/Client/Frontend/Intro/IntroViewController.swift#L373) doing something as `button.accessibilityIdentifier = "turnOnSync.button"